### PR TITLE
Map cristin artikkel fag to nva

### PR DESCRIPTION
**Pull request by Eirik-Eide-Saur**

### CODE REVIEW

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java [hunk 1]
```diff
@@ -138,6 +147,20 @@ private PublicationContext buildPublicationContext() throws InvalidIsbnException
                        .withOpenAccess(false)
                        .build();
         }
+        if (isJournal()) {
+            List<String> isbnList = new ArrayList<>();
+            isbnList.add(extractIsbn());
+            URL theUrl = new URL("https://www.example.com/");
+            return new Journal.Builder()
+                    .withLevel(HARDCODED_LEVEL)
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:155]: HARDCODED_JOURNAL_LEVEL

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java [hunk 2]
```diff
@@ -138,6 +147,20 @@ private PublicationContext buildPublicationContext() throws InvalidIsbnException
...
+                    .withPeerReviewed(HARDCODED_PEER_REVIEWED)
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:156]: HARDCODED_JOURNAL_PEER_REVIEWED

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java (hunk 1 above)
Comment (Eirik-Eide-Saur) [reply cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java:155]: Why cant they be re-used from the Book version?

File: cristin-import/src/main/resources/ignoredFields.txt [hunk 3]
```diff
@@ -7,6 +7,7 @@ modifiedDate
 projects
 publishedDate
 doiRequest
+.owner
```
Comment (axthosarouris) [inline cristin-import/src/main/resources/ignoredFields.txt:10]: owner cannot be empty. it is part of the primary key. inject a value when you create the test data

Comment (axthosarouris) [reply cristin-import/src/main/resources/ignoredFields.txt:10]: There is a method in CirstinObject called hardcodePublicationOwner

**Review by axthosarouris — CHANGES_REQUESTED** [review]

**Review by axthosarouris — APPROVED** [review]

**Review by terjehell — APPROVED** [review]

**Pull request merged**